KhivaCsharp/KhivaCsharp/Regression.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) 2019 Shapelets.io
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using khiva.array;
+
+namespace khiva
+{
+    namespace regression
+    {
+        /// <summary>
+        /// Khiva Regression class containing regression methods.
+        /// </summary>
+        public static class Regression
+        {
+
+            /// <summary>
+            /// Calculates a linear least-squares regression for two sets of measurements. Both arrays should have the same
+            /// length.
+            /// </summary>
+            /// <param name="xss">Expects an input array whose dimension zero is the length of the time series (all the same) and dimension
+            /// one indicates the number of time series.</param>
+            /// <param name="yss">Expects an input array whose dimension zero is the length of the time series (all the same) and dimension
+            /// one indicates the number of time series.</param>
+            /// <returns>Tuple with 
```

Comment:
join these lines: 

            /// <returns>Tuple with 
            /// the slope of the regression line, 
            /// the correlation coefficient,